src/main.coffee

```diff
@@ -114,7 +118,7 @@ app.on 'ready', ->
         height: 590
         "min-width": 620
         "min-height": 420
-        icon: path.join __dirname, 'icons', 'icon@32.png'
+        icon: path.join YAKYAK_ROOT_DIR, 'icons', 'icon@32.png'
```

Comment:
are you sure about this ? is icon a plain path or an url ? I believe a plain path should not be encoded.